From d9fa44dd0fae50990ce2312b08d5cb9898b8edbe Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@freescale.com>
Date: Tue, 28 Jul 2015 18:03:57 +0300
Subject: [PATCH 191/452] dpaa2-eth: Fix compilation of Rx Error FQ code

Conditionally-compiled code slipped between cracks when FLibs were
updated.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@freescale.com>
[Xulin: Original patch taken from FSL LS2085 SDK EAR6.0,
LS2085A-SDK-SOURCE-20160304-yocto.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
index 15bbd82..c4cb865 100644
--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
@@ -1820,14 +1820,14 @@ static int ldpaa_rx_err_setup(struct ldpaa_eth_priv *priv,
 	queue_cfg.dest_cfg.priority = 1;
 	queue_cfg.user_ctx = (uint64_t)fq;
 	queue_cfg.dest_cfg.dest_id = fq->channel->dpcon_id;
-	err = dpni_set_rx_err_queue(priv->mc_io, priv->mc_token, &queue_cfg);
+	err = dpni_set_rx_err_queue(priv->mc_io, 0, priv->mc_token, &queue_cfg);
 	if (unlikely(err)) {
 		netdev_err(priv->net_dev, "dpni_set_rx_err_queue() failed\n");
 		return err;
 	}
 
 	/* Get the FQID */
-	err = dpni_get_rx_err_queue(priv->mc_io, priv->mc_token, &queue_attr);
+	err = dpni_get_rx_err_queue(priv->mc_io, 0, priv->mc_token, &queue_attr);
 	if (unlikely(err)) {
 		netdev_err(priv->net_dev, "dpni_get_rx_err_queue() failed\n");
 		return err;
-- 
1.7.5.4

